packages/puppeteer-core/src/api/ElementHandle.ts

```diff
@@ -886,7 +886,25 @@ export class ElementHandle<
    * or `null` if the element is not visible.
    */
   async boundingBox(): Promise<BoundingBox | null> {
```

Comment:
hm I have just noticed that this is in the wrong file. This would override the CDP implementation in common. Instead this should be defined in common/bidi/ElementHandle where also the sandboxes should be available.

Reply:
scratch the part about overriding the CDP implementation :)

Reply:
It gets overridden in CDP.

Reply:
But yes, the sandbox is available on the BiDi ElementHandle! I've changed the implementation.